lib/xero_gateway/base_record.rb

```diff
@@ -0,0 +1,97 @@
+module XeroGateway
```

Comment:
Yeah, possibly. Although `ActiveModel` doesn't provide the type and attribute *definition* stuff, right?